Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/States/States.cs

```diff
@@ -23,12 +23,30 @@ public class States : ScriptableObject
         /// </summary>
         private static readonly List<Type> candidateStateTypes = new List<Type>() { typeof(InteractableStates) };
 
+        public static States GetDefaultInteractableStates()
+        {
+            States result = new States();
+            List<State> defaultStates = new List<State>();
+
+            InteractableStates allInteractableStates = new InteractableStates();
+            defaultStates.Add(allInteractableStates.GetState(InteractableStates.InteractableStateEnum.Default));
```

Comment:
It would be nice to move the state collection for the default states into a separate function that is virtual, in order to allow someone to override which enums are added to the list in their interactible.